events.go

```diff
@@ -286,26 +286,26 @@ type EventDeploymentFailed struct {
 
 // EventDeploymentInfo describes a 'deployment_info' event.
 type EventDeploymentInfo struct {
-	EventType   string          `json:"eventType"`
-	CurrentStep *DeploymentStep `json:"currentStep"`
-	Timestamp   string          `json:"timestamp"`
-	Plan        *DeploymentPlan `json:"plan"`
+	EventType   string            `json:"eventType"`
+	CurrentStep *DeploymentStepV1 `json:"currentStep"`
```

Comment:
I think we should keep `DeploymentStepV1` as it is, because it is also used to parse [deployments](https://github.com/gambol99/go-marathon/blob/master/deployment.go#L91) for compatibility. We could clone `DeploymentStepV1` to a new struct.

As for the name, how about `DeploymentCurrentStep` ? I think `CurrentStep` is also ok, as this struct is just used in `EventDeploymentInfo`, `EventDeploymentStepSuccess` and `EventDeploymentStepFailure` under events.go and we don't need to tell it's a deployment step.